### PR TITLE
Add New Terrain Tiles

### PR DESCRIPTION
**Pull request by KevDoy**
Discussion happening here.
https://github.com/SFTtech/openage/discussions/1366

This adds the initial free tiles - Grass and Dry Grass.

Cheers

### CODE REVIEW

**Comment by heinezen** [comment]
The `.DS_STORE` file is from macOS isn't it? :D That should be removed.

**Comment by TheJJ** [comment]
Once you track a file with git, adding it to the gitignore doesn't forget it. In the gitignore, you can just write `.DS_Store`.
You you have to use `git rm` to remove the file, and then do a commit. In order to really remove the file (and not just record the deletion in a separate commit), these commits need to be squashed. You can do this with an [interactive rebase](https://github.com/SFTtech/openage/blob/master/doc/contributing.md#rebasing). If you need assistance, please join in our chat :)

**Comment by KevDoy** [comment]
@TheJJ It's been a while... let me know if I did that correctly. Cheers.

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ __pycache__
 CMakeFiles
 cmake_install.cmake
 CMakeCache.txt
+data/terrain/.DS_Store
+.DS_Store
```
Comment (heinezen) [inline .gitignore]: You can just add `**/.DS_Store` I think which should capture all of those files.

**Comment by heinezen** [comment]
Hah what a nice surprise! The `.DS_Store` thing should still be resolved but I think that shouldn't take another 2 years.

**Comment by KevDoy** [comment]
oops. Appreciate the advice there. Should be good to go. If not, let me know.

**Comment by heinezen** [comment]
@KevDoy No problems this time :)

**Pull request merged**